kythe/java/com/google/devtools/kythe/analyzers/java/KytheDocTreeScanner.java

```diff
@@ -98,4 +114,36 @@ public Void visitReference(ReferenceTree tree, DCDocComment doc) {
 
     return null;
   }
+
+  @Override
+  public Void visitDeprecated(DeprecatedTree node, DCDocComment doc) {
+    if (node.getBody().isEmpty()) {
+      // deprecated tag is empty
```

Comment:
Wouldn't the method/class still want to have the deprecated fact emitted even if the tag was empty?

Reply:
The `@Deprecated` annotation is the source of truth for whether the API is deprecated, the javadoc is mostly useful for documentation, and the documentation isn't useful if it's empty. So I'm not sure tools would actually want to surface an empty `@deprecated` tag, but there's no reason not to record it just in case and let it get filtered out later.

I updated the implementation to save empty strings here and added a test.